### PR TITLE
Feature/responses page outline

### PR DESCRIPTION
**Pull request by bryan-robitaille**
# Summary | Résumé

Adds a skeleton page to hold the responses retrieved from the Vault.

Buttons open new modals for confirming responses and reporting a problem.  The functionality and styling for the modals will be added in a separate PR.

Data for the Response Table is fetched on page creation and sent through page props.  A users ability to access the responses is checked on the server before fetching and sending the responses to the browser.

| Before                                          | After                                        |
| ----------------------------------------------- | -------------------------------------------- |
|  NA / Didn't exist|Screenshot 2023-02-14 at 9 41 43 AM |
|  NA / Didn't exist|Screenshot 2023-02-14 at 9 40 17 AM|

# Test instructions | Instructions pour tester la modification

1. Setup and run localstack
2. Submit forms through client
3. Invoke reliability queue script with submission IDs so they are moved to the Vault
4. Navigate to Form page and Responses section in app
5. List of responses in the vault should appear

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

- Functionality and Styling in Modals
- Response Table
- Add default for `lastDownloadedby` if value is undefined in the Vault record

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [x] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: pages/form-builder/responses/[[...params]].tsx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    "whitespace-nowrap relative py-0 px-2 rounded-lg border-2 border-solid inline-flex items-center active:top-0.5 focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500  bg-white-default text-black-default border-black-default hover:text-white-default hover:bg-gray-600 active:text-white-default active:bg-gray-500";
+
+  const dialogConfirmReceipt = useDialogRef();
+  const [isShowConfirmReceiptDialog, setIsShowConfirmReceiptDialog] = useState(false);
+  const dialogConfirmReceiptHandleClose = () => {
+    setIsShowConfirmReceiptDialog(false);
+    dialogConfirmReceipt.current?.close();
+  };
+  const buttonActionsConfirmReceipt = (
+    <Button onClick={dialogConfirmReceiptHandleClose}>{t("responses.confirmReceipt")}</Button>
```
Comment (timarney) [inline pages/form-builder/responses/[[...params]].tsx:36]: Not sure if the style you looking for here lines up but you might be able to pass a theme prop to the button

```
theme?: "primary" | "secondary" | "destructive" | "link" | "icon";
```

File: public/static/locales/en/form-builder.json [hunk 2]
```diff
@@ -329,5 +329,12 @@
   "translateTitle": "Enter your translation",
   "unlockPublishing": "Unlock publishing",
   "unlockPublishingDescription": "To unlock publishing, provide the contact information of the person you report to.",
-  "untitledForm": "Unnamed form file"
+  "untitledForm": "Untitled form",
```
Comment (timarney) [inline public/static/locales/en/form-builder.json:332]: This should be "Unnamed form file" --- looks like that update was reverted here

File: public/static/locales/fr/form-builder.json [hunk 3]
```diff
@@ -329,5 +329,12 @@
   "translateTitle": "Saisir la traduction",
   "unlockPublishing": "Soumettre",
   "unlockPublishingDescription": "Pour déverrouiller la publication, fournissez les coordonnées de la personne de qui vous relevez. ",
-  "untitledForm": "[FR] Unnamed form file"
+  "untitledForm": "[FR] Untitled form",
```
Comment (timarney) [inline public/static/locales/fr/form-builder.json:332]: This should be reverted to "[FR] Unnamed form file"

File: public/static/locales/en/form-builder.json (hunk 2 above)
Comment (bryan-robitaille) [reply public/static/locales/en/form-builder.json:332]: Thanks will fix that merge conflict

File: public/static/locales/fr/form-builder.json (hunk 3 above)
Comment (bryan-robitaille) [reply public/static/locales/fr/form-builder.json:332]: Thanks will fix that merge conflict

File: pages/form-builder/responses/[[...params]].tsx (hunk 1 above)
Comment (bryan-robitaille) [reply pages/form-builder/responses/[[...params]].tsx:36]: As I mentionned in the PR the styling of the modals was out of scope.  I just grabbed what Pete has started and all that styling will get sorted in the next PR.

Comment (bryan-robitaille) [reply pages/form-builder/responses/[[...params]].tsx:36]: Yeah, I have yet to clean up all the styling going on in this page.  
Will get to it before I convert to ready for review.

Comment (thiessenp-cds) [reply pages/form-builder/responses/[[...params]].tsx]: Done. I tried to re-use the Form-builder buttons (and other components) as much as possible. One nav item is a link, that is styled to look like a button

**Review by craigzour — APPROVED** [review]
LGTM :)

**Pull request merged**